tests/market_data/test_market_data_api.py

```
@@ -72,6 +81,142 @@ def test_create_data_ingestion(monkeypatch):
+    assert calls == [{"include_tpex": False, "reconcile": False}]
+    assert response.json()["inactivated_count"] == 1
+    assert response.json()["reconciliation_requested"] is False
+    assert response.json()["reconciliation_skipped"] is True
+    assert response.json()["source_summaries"][0]["exchange"] == "TWSE"
+
+
+def test_tw_company_crawl_does_not_expose_request_url(
+    caplog,
+    monkeypatch,
+):
+    secret_url = "https://feed.test/company?token=secret"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "secret_url"

(S105)